### PR TITLE
Recognisiton using cnn

### LINKED ISSUE #8: Recognition using CNN
Facial Recognition using Convolutional Neural Networks

### PR DESCRIPTION
**Pull request by Halix267**
closes #8

### CODE REVIEW

**Comment by Halix267** [comment]
@akshitagupta15june  Plzz review it ,I had made the changes

**Comment by akshitagupta15june** [comment]
great work @Halix267 can you tell me the process of running this cnn pr
like which code should i run in which order to check your results.

**Comment by akshitagupta15june** [comment]
So that i can have a look at your output

**Comment by akshitagupta15june** [comment]
and @Halix267 can you also create a readme file like what steps you have performed and what output you have got.
So that if anyone will have a look at this they will get to know whole process as well as output.

**Comment by Halix267** [comment]
u can jst cpy the cde in your vsc or colab and create the dataset

**Comment by Halix267** [comment]
the  just train the model  and whn the training complete  a file name final_model.py create on yr the folder where the the images save than run displaying resut.py

**Comment by Halix267** [comment]
So basically the order is Create dataset.py , Training the model, Displaying results

**Comment by Halix267** [comment]
And one mre thing bfre traing the model  Split the dataset into train and test set , Basically I am saying that u should split the images into to folders Train and Test than u go for training

**Comment by akshitagupta15june** [comment]
Can you send me working of this feature like any screenshots or something

**Comment by Halix267** [comment]
ok

**Comment by Halix267** [comment]
Screenshot from 2020-12-11 17-58-37

**Comment by Halix267** [comment]
Screenshot from 2020-12-11 17-59-00

**Comment by Halix267** [comment]
This is the output @akshitagupta15june . Is this is fine

**Comment by akshitagupta15june** [comment]
It is perfect 👍

**Comment by Halix267** [comment]
So any changes needed?

**Comment by Halix267** [comment]
and how much points do i get for this issue :)

**Comment by akshitagupta15june** [comment]
You will get 15+5 =20 points

**Comment by Halix267** [comment]
Can I work on other face recognition models also??

**Comment by akshitagupta15june** [comment]
@Halix267 I am merging this pull request.
It will be great if you also add a readme in this like the process you did while training and testing using CNN as this will be informative for fellow students.

**Comment by akshitagupta15june** [comment]
I have merged your pr and updated your points.

**Comment by akshitagupta15june** [comment]
> Can I work on other face recognition models also??

Yes ofcourse you can work

**Comment by Halix267** [comment]
@akshitagupta15june but it is not shown in leaderboard. How much points u gave me?

**Comment by akshitagupta15june** [comment]
your points were 5 and i incremented it to 25

**Comment by Halix267** [comment]
no that not my points @akshitagupta15june  look above in leaderboard I have 40 points

**Comment by akshitagupta15june** [comment]
I hope u got it

**Comment by Halix267** [comment]
there is problem in lederboard , The leaderboard showing my  previous points also @akshitagupta15june . Plzz update the points where I have 40 points in leaderboard

**Comment by akshitagupta15june** [comment]
In leadeboard i can see 5 pints only.
I only have rights to update your points which are in this projects.
For that increment you have to contact other project admins to update.

**Comment by Halix267** [comment]
Screenshot from 2020-12-11 19-14-57

**Comment by Halix267** [comment]
See @akshitagupta15june

**Comment by akshitagupta15june** [comment]
ok @Halix267 i am updating it 65 means 40+25.

**Comment by akshitagupta15june** [comment]
@Halix267 can u check is it updated to 60?

**Comment by Halix267** [comment]
wait

**Comment by akshitagupta15june** [comment]
or i have updated it in the sheet maybe it will take time.

**Comment by Halix267** [comment]
Yes I think so bcz it is not shown yet

**Comment by akshitagupta15june** [comment]
i have checked just now its updated to 60.
Good work @Halix267

**Comment by Halix267** [comment]
Also I have told raised this issue in nwoc gitter channel also . May be they do something about my name showing twice in leaderborad bcz it confuses the mentors

**Comment by akshitagupta15june** [comment]
yes your name is twice

**Comment by Halix267** [comment]
All thanks to you @akshitagupta15june :)

**Comment by akshitagupta15june** [comment]
Its your efforts!!
